src/test/java/kitchenpos/builder/OrderLineItemBuilder.java

```diff
@@ -0,0 +1,43 @@
+package kitchenpos.builder;
+
+import kitchenpos.model.OrderLineItem;
+
+public class OrderLineItemBuilder {
+    private Long seq;
+    private Long orderId;
+    private Long menuId;
+    private long quantity;
+
+    public OrderLineItemBuilder() {
+    }
+
+    public OrderLineItemBuilder seq(Long val) {
+        seq = val;
+        return this;
+    }
+
+    public OrderLineItemBuilder orderId(Long val) {
+        orderId = val;
+        return this;
+    }
+
+    public OrderLineItemBuilder menuId(Long val) {
+        menuId = val;
+        return this;
+    }
+
+    public OrderLineItemBuilder quantity(long val) {
+        quantity = val;
+        return this;
+    }
+
+    public OrderLineItem build() {
+        final OrderLineItem oderLineItem = new OrderLineItem();
```

Comment:
중요한 내용은 아니지만 오타가 있네요!

```suggestion
        final OrderLineItem orderLineItem = new OrderLineItem();
```